doc/source/advanced-use.rst

```diff
@@ -11,251 +16,317 @@ Essentially, clones are cheap, lightweight, and writable snapshots.
+Check the currently active limits on a jail with:
 
-:samp:`# iocage limits UUID | TAG`
+:command:`iocage limits [UUID | TAG]`
 
-Limit CPU use by %
-++++++++++++++++++
+.. index:: Limit CPU Usage by Percentage
+.. _Limit CPU Usage by Percentage:
 
-Limit CPU execution to 20%:
+Limit CPU Usage by %
+++++++++++++++++++++
+
+In this example, :command:`iocage` limits *testjail* CPU execution to
+20%, then applies the limitation to the active jail:
 
-:samp:`# iocage set pcpu=20:deny UUID|TAG`
-:samp:`# iocage cap UUID|TAG`
+:samp:`# iocage set pcpu=20:deny testjail`
+:samp:`# iocage cap testjail`
 
-Check limits:
+Double check the jail's current limits to confirm the functionality:
 
-:samp:`# iocage limits UUID | TAG`
+:samp:`# iocage limits testjail`
 
-Resetting a jail's properties
-+++++++++++++++++++++++++++++
+.. index:: Reset Jail Properties
+.. _Reset Jail Properties:
 
-iocage easily allows resetting a jail's properties back to their
+Reset Jail Properties
++++++++++++++++++++++
+
+:command:`iocage` easily allows resetting jail properties back to the
 defaults!
 
 To reset to defaults:
 
-:samp:`# iocage reset UUID | TAG`
+:command:`iocage reset [UUID | TAG]`
 
 You can also reset every jail to the default properties:
 
-:samp:`# iocage reset ALL`
+:command:`iocage reset ALL`
+
+Resetting a jail retains the jail's UUID and TAG. All other non-standard
+settings are lost. Be sure to reset any needed custom properties. If
+anything is set by :command:`iocage set [PROPERTY] default`, there is
+nothing else required!
 
-Resetting a jail retains the jail's UUID and TAG. Everything else is
-lost. Be sure to reset any needed custom properties. If anything is set
-by ``iocage set PROPERTY default``, there is nothing else required!
+.. index:: Automatic Package Installation
+.. _Automatic Package Installation:
 
-Automatic package installation
+Automatic Package Installation
 ------------------------------
 
 Packages can be installed automatically at creation time!
 
-Specify the ``pkglist`` property at creation time, which needs to point
-to a text file containing one package name per line. Note an Internet
-connection is required, as :command:`pkg install` obtains packages from
-online repositories.
+Use the [-p | --pkglist] option at creation time, which needs to point
+to a JSON file containing one package name per line.
 
-**Example:**
+.. note:: An Internet connection is required for automatic package
+   installations, as :command:`pkg install` obtains packages from online
+   repositories.
 
-Create a :file:`pkgs.txt` file and add package names to it.
+Create a :file:`pkgs.json` file and add package names to it.
 
-:file:`pkgs.txt`:
+:file:`pkgs.json`:
 
 .. code-block:: none
 
    nginx
    tmux
 
-Now, create a jail and supply :file:`pkgs.txt`:
+Now, create a jail and supply :file:`pkgs.json`:
 
-:samp:`# iocage create pkglist=/path-to/pkgs.txt tag=myjail`
+:command:`iocage create -r [RELEASE] -p [path-to/pkgs.json] -n [NAME]`
```

Comment:
Should note that `--name` is not required, a UUID jail will be created otherwise.